demo/config/default.js

```diff
@@ -0,0 +1,16 @@
+const development = {
```

Comment:
I wonder whether we want to pick `default` or `development` for both the filename and the object?

Reply:
Done. I think that maybe is desirable have more than one file inside `config/` (e.g.: development.js, production.js) and the app will only look for `default.js`. Make sense?

Reply:
Sure, that works for me! Let's just not put any of our API keys in production.js. We can have those injected during the build process in CI.

Reply:
Resolving this